intl/en/messages.json

```diff
@@ -0,0 +1,64 @@
+{
+
+    "Invalid usage (near option `{option}`), try {{`{program} --help`}}.":
```

Comment:
I quite like how you hash the keys to make them acceptable to watson. Why don't you hash the keys here? Its still easy to lookup by value, you just have to invert the object on load (`for every k,v in message: message[v] = k;`).